.ci/linux-steps.yaml

```diff
@@ -49,7 +49,7 @@ steps:
   displayName: 'Build models'
 
 # Run CTests.
-- script: cd build/tests/ && sudo CTEST_OUTPUT_ON_FAILURE=1 ctest -T Test .
+- script: cd build/bin/ && ./models_test -x 50
```

Comment:
Maybe we should leave off the `-x 50`?  That way the log will contain the logs for all failed tests.  (But maybe there is a reason to add it that I overlooked?)

Reply:
So the -x 50 is included because we would want catch2 to continue even if some tests fail. And suppose you have a couple of tests failing therefore 50 is a big enough number to accomodate them all so you can address them at once.

Reply:
Got it.  Thanks for the clarification!

Reply:
No problem 😃